src/Libraries/PythonNodeModelsWpf/TabFoldingStrategy.cs

```diff
@@ -10,7 +10,7 @@ namespace PythonNodeModelsWpf
     /// Allows producing tab based foldings
     /// https://stackoverflow.com/questions/47224064/avalonedit-foldingstrategy-by-indent-python/47577500#47577500
```

Comment:
is this comment still valid?

Reply:
Yes, this code is still largely lifted from there.